new.go

```diff
@@ -202,6 +202,16 @@ func newBuilder(store storage.Store, options BuilderOptions) (*Builder, error) {
 	if img != nil {
 		if len(img.Names) > 0 {
 			image = img.Names[0]
+			// When the image used by the container is a tagged image
```

Comment:
Create a function called getImageName and move this code to it, that will fix your tests failures.

Reply:
Yep, I did it and run the `make validation`. I just want to add some test(s) to that change too.
Thanks for the review, though.